charts/dapr/README.md

```diff
@@ -141,6 +141,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | `dapr_placement.cluster.logStoreWinPath`       | Mount path for persistent volume for log store in windows when HA is true                                                                                                 | `C:\\raft-log`          |
 | `dapr_placement.volumeclaims.storageSize`      | Attached volume size                                                                                                                                                      | `1Gi`                   |
 | `dapr_placement.volumeclaims.storageClassName` | storage class name                                                                                                                                                        |                         |
+| `dapr_placement.maxApiLevel`                  | Sets the `max-api-level` flag which prevents the Actor API level for going above this value. The default value of -1 means no cap.  | `-1` |
```

Comment:
we know that the placement server is only for actors, but should the parameter name still be made clear? `dapr_placement.maxActorApiLevel`

Reply:
Makes sense, changing that.

Reply:
Done